frontend/src/routes/+page.svelte

```diff
@@ -2,25 +2,21 @@
 	import Header from "../lib/components/Header.svelte";
 </script>
 
-<Header title="Looney"/>
-<div class="bg-red-300 py-5 px-5 rounded-lg place-self-center">
-	<form class="grid grid-cols-1 gap-8">
+<Header title="Looney" displayBackButton=0 path="" route="" />
```

Comment:
Not fan of passing empty parameters, can you check if you can set default value of path, route, displayBackButton to empty withing the Header component instead of sending empty data?